docs/apm/filters.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[filters]]
+=== Filters
+
+Global filters are ways you can filter your data within the APM UI.
+They are available in the Services, Transactions, Errors, Metrics, and Traces views,
+and any filter applied will persist as you move between pages.
+
+[role="screenshot"]
+image::apm/images/global-filters.png[Global filters available in the APM UI in Kibana]
+
+[float]
+==== Global time range
+
+The <<set-time-filter,global time range filter>> in {kib} restricts APM data to a specific time period.
+
+[float]
+[[query-bar]]
+==== Query bar
+
+The query bar is a powerful data query feature.
+Similar to the query bar in {kibana-ref}/discover.html[Discover],
+it enables you to pass advanced queries on your data to filter on particular pieces of information that you're interested in.
+It comes with a handy autocomplete that helps find the fields and even provides suggestions to the data they include.
+You can select the query bar and hit the down arrow on your keyboard to begin seeing recommendations.
+
+See <<query-bar,advanced queries>> for more information and sample queries.
+
+[float]
+[[environment-selector]]
+==== Environment selector
```

Comment:
I'd probably rename this to the "Service environment filter" instead of selector. To me it selector sounds like the UI component rather than what the component does.